flat/MintableERC1155Predicate.sol

```diff
@@ -1363,6 +1369,9 @@ contract Initializable {
 pragma solidity 0.6.6;
 
 
+    ERC1155Receiver
+} from "@openzeppelin/contracts/token/ERC1155/ERC1155Receiver.sol";
+
```

Comment:
```suggestion
```
`ERC1155Receiver` is already implemented inside the flattened contract, so this can be removed in its entirety

Reply:
It's autogenerated, I did a clean build and the output is same.